libp2p/Host.cpp

```diff
@@ -850,6 +844,19 @@ bytes Host::saveNetwork() const
 
     RLPStream network;
     int count = 0;
+    if (auto nodeTable = this->nodeTable())
```

Comment:
Changed the order in network.rlp - first node table contents are saved, then connected peers. Otherwise, as peers are probably present in the node table, too, adding them first would always initiate ping.

Reply:
This looks like a copy of the table.

Reply:
It was copy of a `shared_ptr`, but I'll change it to access member variable directly